Add font weight to filament::link component (#13665)
laravel-filament/filament: packages/support/docs/09-blade-components/02-link.md

</x-filament::link>
```

## Setting the font weight of a link

> By default, the font weight of a link is "semi-bold". You can make it "thin", "light", "medium", "semibold", "bold" or "black" by using the `weight` attribute:

Reviewer: ```suggestion
By default, the font weight of links is `semibold`. You can make it `thin`, `extralight`, `light`, `normal`, `medium`, `bold`, `extrabold` or `black` by using the `weight` attribute:
```

```blade
<x-filament::link weight="thin">
    New user
</x-filament::link>

<x-filament::link weight="extralight">
    New user
</x-filament::link>

<x-filament::link weight="light">
    New user
</x-filament::link>

> <x-filament::link weight="medium">

Reviewer: ```suggestion
<x-filament::link weight="normal">
    New user
</x-filament::link>

<x-filament::link weight="medium">
```

    New user
</x-filament::link>

<x-filament::link weight="semibold">
    New user
</x-filament::link>
   
<x-filament::link weight="bold">
    New user
</x-filament::link>

<x-filament::link weight="black">
    New user
</x-filament::link> 
```

> Or enter a custom weight

Reviewer: ```suggestion
Alternatively, you can pass in a custom CSS class to define the weight:
```
